### ISSUE TITLE
Windows 11 Taskbar Styler Mod, How can I make the Taskbar font SemiBold?

### ISSUE THREAD

**Issue by Im-Baked** [ramensoftware/windows-11-taskbar-styling-guide#149]
Hello again, I asked on your main website and you replied, but I'm not sure where to put the information, it keeps giving me errors.

I use a font named named Eurostile Extended, and I'm not sure how or where to write what's needed into the Windhawk app.

There is no option to make it **SemiBold** like there is on the Taskbar Clock Customization Mod.

Windows 11 Taskbar Styler Settings

Control styles
Target
TextBlock#LabelControl
Styles
FontFamily=Eurostile Extended

Windows 11 Taskbar Styler Advanced Settings

Mod settings
Easily export or share your mod settings.
{"theme":"","controlStyles[0].target":"TextBlock#LabelControl","controlStyles[0].styles[0]":"FontFamily=Eurostile Extended","resourceVariables[0].variableKey":"","resourceVariables[0].value":""}

Where exactly do I put the info that's needed to make it **SemiBold**? Thanks for any help with the issue!

A couple screenshots
https://i.postimg.cc/kJ1HNrLL/Settings.jpg
https://i.postimg.cc/hnpNRxhM/Advanced.jpg

**Comment by m417z** [comment]
No need to use the advanced settings in your case, just add a new style in settings after FontFamily, and set it to "FontWeight=Bold".

**Comment by Im-Baked** [comment]
Thanks for the reply, unfortunately that makes it go back to the default font.. [https://i.postimg.cc/jTFvhhDV/Clipboard-1.jpg](url)

**Comment by m417z** [comment]
Click on "Add new item", then have two entries:
* `FontFamily=Eurostile Extended`
* `FontWeight=Bold`

**Comment by Im-Baked** [comment]
Alright I did that and it still is the same, does it look right on the settings on this pic? https://i.postimg.cc/9CGpC5JV/Clipboard-12.jpg

**Comment by m417z** [comment]
Remove the quotes.

**Comment by Im-Baked** [comment]
Oh my it actually works and looks alot better, thanks so mcuh! https://i.postimg.cc/TdLVsZpz/Clipboard-12-15-2024-02.jpg

I saw on your site about donating, can it be done with paypal?

**Comment by Im-Baked** [comment]
I don't have that Bitcoin, Yes I just looked on your site and see Paypal there

**Comment by m417z** [comment]
Yes, it can be done with PayPal. Thanks for considering a donation!

**Comment by Im-Baked** [comment]
This is great, the default font in win 11 is so small and shitty and can barely read it lol

**Comment by Im-Baked** [comment]
I'm going to recommend this app to everyone I know that is having issues with the shitty windows 11 fonts, I know at least 25 people that hate the small font that it has by default, it can be made bigger in windows with that Text resizer app it has built in, but that makes everything on the whole system have large fonts, and it makes some wayyy to big. Awesome thanks again for making such a wonder program, you should put that in as an option for the taskbar mod to make it have bolder fonts, would make it so much easier for people like me that have no idea where to put the info at 1st. Until they get get help with it luckily like I did

**Comment by Im-Baked** [comment]
Thanks again, you have yourself a wonderful night Michael
